### ISSUE TITLE
NoMethodError: undefined method `[]' for nil:NilClass

### ISSUE THREAD

**Issue by fsanggang** [martijn/xsv#40]
I'm using xsv v1.1.0, and trying to open [this file](https://github.com/martijn/xsv/files/8358701/minimal_example.xlsx).

The error I get is:
```
Xsv::Workbook.open(path_to_file)
NoMethodError: undefined method `[]' for nil:NilClass
from /Users/fionasanggang/.asdf/installs/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/xsv-1.1.0/lib/xsv/sheet.rb:36:in `initialize'
```

I believe this document was generated using the [Open XML SDK 2.5 for Office](https://docs.microsoft.com/en-us/office/open-xml/structure-of-a-spreadsheetml-document#generated-spreadsheetml), due to the` <x:worksheet>`, `<x:sheetData>` elements in the sheet.xml file.

Do you have any hints or suggestions?

**Comment by martijn** [comment]
Thanks for sharing an example! I've identified two issues with this file so far:

- The workbook.xml uses valid but non-standard namespaces, which the XML parser doesn't account for
- The workbook.xml.rels  has a leading / on the Target attribute which other files don't have

Even after resolving those two, there are still no contents returned for the sheet. I assume this also because of XML namespaces. The XML parser is not aware of namespaces because so far every file I've encountered uses the same as standard Office. I will fix this as soon as I can find the time.

**Comment by martijn** [comment]
@fsanggang I fixed the case for this test file in https://github.com/martijn/xsv/pull/41, could you see if the `issue-40` branch resolves all your issues before I merge and release that?

**Comment by fsanggang** [comment]
Thank you for the quick response @martijn, much appreciated! I will check `issue-40` and get back to you asap.

**Comment by fsanggang** [comment]
@martijn I can confirm your changes in https://github.com/martijn/xsv/pull/41 works and resolves all my issues. Once again - many thanks!

**Comment by martijn** [comment]
You're welcome! Released Xsv 1.1.1 with the fix.
